style.css

```diff
@@ -1,65 +1,54 @@
 body {
-    padding: 0;
-    margin: 0;
-    font-family: Arial, Helvetica, sans-serif;
-    background-color: rgb(226, 226, 226);
-}
-
-header {
-    text-align: center;
-}
-
-div.todo {
-    padding: 1em;
-}
+  padding: 0;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #e2e2e2; }
 
 .container {
-    max-width: 25em;
-    margin: 0 auto;
-    background-color: white;
-    box-shadow: 0 0 10px rgb(172, 172, 172);
-}
-
-/* New Todo Field */
-#new-todo {
+  max-width: 25em;
+  margin: 0 auto;
+  background-color: white;
+  box-shadow: 0 0 10px #acacac; }
+
+.todo {
```

Comment:
So one of the things web BEM is that you want to avoid nesting in most cases. When you do nesting you can end up with really weird specificity issues. Nesting also couples your CSS directly to the structure of the DOM, and so if you change the DOM structure, your CSS kind of falls apart.